cssgunc/CSPortal
# [Footer] Added basic footer

## Summary 

Rendered a basic footer with social media and website links.

## To test 

1. Check out the sample-pull-request branch `git checkout footer`
2. Run `npm start` in your command line inside the RTCPortal root directory
3. See the footer

## Test cases 

**1. Super short page:**

![Screen Shot 2020-10-11 at 6 27 45 PM](https://user-images.githubusercontent.com/43911326/95691867-96809d80-0bf0-11eb-8047-eb501899ce05.png)

**2. Full page with links (see the bottom corners for the hyperlinks):**

![footertest](https://user-images.githubusercontent.com/43911326/95691871-9da7ab80-0bf0-11eb-9005-1ec4666f9da3.gif)

## Next steps:
1. Move social media account links and icons to Airtable
2. Move website and mail links to Airtable
